ledger/src/blockstore_processor.rs

```diff
@@ -327,21 +327,17 @@ fn execute_batches(
 
     let mut minimal_tx_cost = u64::MAX;
     let mut total_cost: u64 = 0;
-    let mut total_cost_without_bpf: u64 = 0;
     // Allowing collect here, since it also computes the minimal tx cost, and aggregate cost.
     // These two values are later used for checking if the tx_costs vector needs to be iterated over.
-    // The collection is a pair of (full cost, cost without estimated-bpf-code-costs).
     #[allow(clippy::needless_collect)]
     let tx_costs = sanitized_txs
         .iter()
         .map(|tx| {
             let tx_cost = CostModel::calculate_cost(tx, &bank.feature_set);
             let cost = tx_cost.sum();
-            let cost_without_bpf = tx_cost.sum_without_bpf();
```

Comment:
Question for potential follow-up: this was the only external use of `sum_without_bpf()`.

Should we consider making the fn not `pub`, or even move the functionality into `sum()`?

Reply:
Ah yes! Will remove `sum_without_bpf` in a separate PR.